### ISSUE TITLE
Doesn't work with bevy 0.6.0

### ISSUE THREAD

**Issue by GrbavaCigla** [JonahPlusPlus/bevy_atmosphere#3]
This is my Cargo.toml:
```toml
[package]
name = "projectname"
version = "0.1.0"
edition = "2021"

[dependencies]
bevy = "0.6.0"
bevy_atmosphere = "0.1.3"
```
And this is the error I get when running `cargo run`:
```bash
$ cargo run
    Updating crates.io index
error: failed to select a version for `syn`.
    ... required by package `darling_core v0.13.0`
    ... which satisfies dependency `darling_core = "=0.13.0"` of package `darling v0.13.0`
    ... which satisfies dependency `darling = "^0.13"` of package `ndk-macro v0.3.0`
    ... which satisfies dependency `ndk-macro = "^0.3.0"` of package `ndk-glue v0.5.0`
    ... which satisfies dependency `ndk-glue = "^0.5"` of package `bevy_asset v0.6.0`
    ... which satisfies dependency `bevy_asset = "^0.6.0"` of package `bevy_audio v0.6.0`
    ... which satisfies dependency `bevy_audio = "^0.6.0"` of package `bevy_internal v0.6.0`
    ... which satisfies dependency `bevy_internal = "^0.6.0"` of package `bevy v0.6.0`
    ... which satisfies dependency `bevy = "^0.6.0"` of package `projectname v0.1.0 ([my project path])`
versions that meet the requirements `^1.0.69` are: 1.0.86, 1.0.85, 1.0.84, 1.0.83, 1.0.82, 1.0.81, 1.0.80, 1.0.79, 1.0.78, 1.0.77, 1.0.76, 1.0.75, 1.0.74, 1.0.73, 1.0.72, 1.0.71, 1.0.70, 1.0.69

all possible versions conflict with previously selected packages.

  previously selected package `syn v1.0.65`
    ... which satisfies dependency `syn = "=1.0.65"` of package `bevy v0.5.0`
    ... which satisfies dependency `bevy = "^0.5"` of package `bevy_atmosphere v0.1.3`
    ... which satisfies dependency `bevy_atmosphere = "^0.1.3"` of package `projectname v0.1.0 ([my project path])`

failed to select a version for `syn` which could resolve this conflict
```

**Comment by JonahPlusPlus** [comment]
Yeah, this is something I've been meaning to update. I can't promise anything right now. 0.6 has made a lot of changes to the renderer, and I have yet to dive into the code to figure it out.

**Comment by kbieganski** [comment]
I was just working on this. See https://github.com/JonahPlusPlus/bevy_atmosphere/pull/4

**Comment by JonahPlusPlus** [comment]
Fixed with 0.2.0.
